generators/entity/index.js

```diff
@@ -62,8 +62,21 @@ module.exports = yeoman.generators.Base.extend({
     // don't prompt if data are imported from a file
     if (this.entityConfig.useConfigurationFile == true &&  this.entityConfig.data && typeof this.entityConfig.data.enableEntityAudit !== 'undefined') {
       this.enableAudit = this.entityConfig.data.enableEntityAudit;
+
+      if (ypeof this.entityConfig.data.auditFramework !== 'undefined') {
+        this.auditFramework = this.entityConfig.data.auditFramework;
```

Comment:
Wasnt this saved in yo-rc file in main generator? Then it should be read grom config